domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -690,7 +700,8 @@ class TopicListController @Inject constructor(
     totalChapterCount: Int,
     nextChapterName: String?,
     explorationId: String?,
-    isTopicConsideredCompleted: Boolean
+    isTopicConsideredCompleted: Boolean,
+    chapterProgress: ChapterProgress?
```

Comment:
Can this name be more specific? Maybe ``nextChapterProgress``?

Reply:
updated name.